chainer/utils/experimental.py

```diff
@@ -0,0 +1,40 @@
+import chainer
+import inspect
+import warnings
+
+
+def experimental(api_name=None):
+    """A function for marking APIs as experimental
+
+    Developer of the API can mark it as *experimental* by calling
+    this function. When users call experimental APIs, `FutureWarning`
+    is raised once for each experimental API.
+    The presentation of `FutureWarning` is disabled by setting
+    `chainer.disable_experimental_warning` to `True`,
+    which is `False` by default.
+
```

Comment:
Can this also be used to mark a class as experimental? What is the suggested way to do that? Should I call `utils.experimental()` from inside the `__init__()` initializer? If so, please consider adding documentation that explains and/or gives examples of how to use the experimental feature to mark an API as experimental at the different granularities (function/method, class, module).